common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/CurrentTaskAuthorizationFragment.java

```diff
@@ -121,7 +121,7 @@
      * Initialize based on value provided in intent.
      */
     private static String setDiagnosticContextForNewThread(@NonNull final String correlationId) {
-        final String methodName = ":setDiagnosticContextForAuthorizationActivity";
+        final String methodName = "#setDiagnosticContextForAuthorizationActivity";
```

Comment:
this was correct, dont need to change

Reply:
Here: da1e62d1ddacd758eb5d0a6cedd9589521475ab8